### PR TITLE
CLN: Type error fix in tests\tseries\offsets\test_yqm_offsets.py

### PR DESCRIPTION
**Pull request by lukasbk**
- [ ] xref #28926
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/extension/base/base.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+
 import pandas.util.testing as tm
 
 
 class BaseExtensionTests:
 
     assert_equal = staticmethod(tm.assert_equal)
-    assert_series_equal = staticmethod(tm.assert_series_equal)
-    assert_frame_equal = staticmethod(tm.assert_frame_equal)
+    assert_series_equal = staticmethod(tm.assert_series_equal)  # type: Any
```
Comment (jreback) [inline pandas/tests/extension/base/base.py]: why is this needed?

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/extension/base/base.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/extension/base/base.py]: I'm not clear on why these need to be staticmethods at all

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
@lukasbk can you merge master and keep the changes from #28994 and #28990 seperate.

**Review by simonjayhawkins — COMMENTED** [review]
@lukasbk Thanks for updating the PR. will need to merge master once #29010 merged.

**Comment by lukasbk** [comment]
Alright! Sorry for the mess. Still learning how this git really works ;)

**Comment by TomAugspurger** [comment]
Merging master should fix the CI errors.

**Comment by lukasbk** [comment]
Did I do something wrong?

**Comment by jbrockmendel** [comment]
> Did I do something wrong?

No, the 3.8 build is just flaky.  I restarted it, so hopefully this will be green in a bit.

**Review by simonjayhawkins — APPROVED** [review]
@lukasbk lgtm @jreback @WillAyd

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Great thanks @lukasbk
